libraries/src/Toolbar/Toolbar.php

```diff
@@ -53,12 +54,12 @@ class Toolbar
 	protected $_name = [];
 
 	/**
-	 * Toolbar array
+	 * Toolbar array, we keep track of buttons in specific sections.
 	 *
 	 * @var    array
 	 * @since  1.5
 	 */
-	protected $_bar = [];
+	protected $_bar = array(array());
```

Comment:
@euismod2336 Why not use short array syntax here?

Reply:
can't init array in array with short syntax, maybe array([]) would work but that seems less readable

Reply:
I kinda figured but wanted to be sure. In that case, keep it as it is.